Makefile

```diff
@@ -58,7 +58,7 @@ build: test
 	 "
 	 @rm -f ./build/embeddedResources.zip
 	 @cd ./commands && zip -q -r ../build/embeddedResources.zip ./bootstrap/resources/*
-	 @cd ./data && zip -q -r ../build/embeddedResources.zip ./*
+	 @cd ./data && zip -q -r ../build/embeddedResources.zip ./* -x tosca/test-types.yml
```

Comment:
TIL that the `data/tosca` definitions get added to the binary and stored to consul during startup. This ignores the test-types during zip compression so they don't get added.

Reply:
Thanks ! Good to know !

Reply:
I do not get the point on having this for testing only :thinking: 
If it is for testing it is probably not the right place